style.css

```diff
@@ -25,14 +31,56 @@ html {
 
 .main-header {
   height: 5rem;
-  padding: 24px;
+
+  /* padding: 24px; */
   background-color: #fff;
```

Comment:
- [ ] `Required`
- [ ] please make sure make the space on top of the navbar same as the original template

`your template`
![image](https://user-images.githubusercontent.com/66230306/231138272-9f65487f-de82-4c8f-bcc4-adf779541a40.png)

`figma tempalte`
![image](https://user-images.githubusercontent.com/66230306/231138862-0e65f81a-4549-4ff5-b1db-0d793864cff9.png)